### PR TITLE
movie: 2067 (2020) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
A stinker. Rating 1/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -356,3 +356,46 @@ demon [Bill and Ted met a pair of rather friendly demons during their brief stay
...
+destiny [Ethan was convinced that it was his fate (a word that is uttered five times in the film) that Jude would shoot him dead, and in so doing put an end to Ethan's effort to save humanity. Toward the end, however, Ethan changed his tune an resolved to change his own fate, and the fate of humanity by extension.]
+father and son [Central to the story was Ethan's dealing with his daddy issues.]
+
+:: Minor Themes
+coping with being in a heat wave [A reporter spoke of a record-breaking heat wave.]
+hologram [People communicated via holographic recordings.]
+suicide [A guilt-ridden Jude took his own life. It was said that Ethan's father had committed suicide when Ethan was only eight years old. Although it was later revealed that the Chronicorp CTO had shot him dead.]
+love vs. the public good [Should Ethan stay in the present to care for his gravely ill wife or go on a possibly one way trip into the future in a last ditched effort to save humanity?]
+wilderness survival [Ethan rubbed two sticks together to make a fire to survive the night in the rainforest.]
+personal identity [Ethan was puzzled about if his future skeleton was him.] {48:30}
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:393]: I guess this {} time mark should not be merged into master?

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -356,3 +356,46 @@ demon [Bill and Ted met a pair of rather friendly demons during their brief stay
...
+cold fusion [The handheld computer Archie detected a corrupted cold fusion power feed in the time machine's nuclear core.]
+mother and son [A young boy watched his mother die of a gunshot wound to the neck.]
+remorse [a guilt-ridden Ethan spoke of how he'd left his wife to die. Richard lamented that he hadn't been there for his son Ethan.]
+parental love [Richard pointedly proclaimed his love to his son Ethan.]
+choosing who lives and who dies [Regina Jackson selected the "chosen few" who would be sent 407 years into the future to reboot civilization.]
+time portal [The prototype time machine worked by generating a portal of some sort that one could pass through.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:399]: I thought these things ought to be more or less permanent fixtures to qualify - our Wikipedia reference concurs with me though I see our own definition text could use a little TLC.

Comment (odinlake) [reply notes/stories/film/film-scifi-2020s.st.txt:399]: At any rate I really think we should find a way not to use it when it's merely a time machine that briefly opens up a portal like opening. Usually that's just as convenient visual effect and doesn't make the machine different from any other time machine.

**Review by odinlake — COMMENTED** [review]
left comments in diff

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Whoops. That's one of my time cues to remind me to go back and figure out what was going on later. Neglected to erase that one before my initial commit. It's gone now.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Very good. I took it out. I was annoyed that I felt compelled by our definition to use it in the first place. I'll submit a new pull request with a revised version of the theme.

**Pull request merged**